data-connector-lib/src/dpk_connector/core/crawler.py

```diff
@@ -74,6 +74,7 @@ def async_crawl(
     user_agent: str = "",
     headers: dict[str, str] = {},
     allow_domains: Collection[str] = (),
+    subdomain_focus: bool = False,
```

Comment:
Should the default be set to True? Its is very rare that we need to do a common crawl of the web and not be restricted/focused on a specific domain or a specific URL...

Reply:
If we set it true by default, the default behavior the crawler would change. We should not introduce any braking change except for major version updates.

Reply:
ok. that makes sense. I have the feeling that for most of the RAG use cases we will be setting this to true.